doc/cephfs/quota.rst

```diff
@@ -1,10 +1,53 @@
-Quotas
-======
+CephFS Quotas
+=============
 
-CephFS allows quotas to be set on any directory in the system.  The
+CephFS allows quotas to be set on any directory in the file system.  The
 quota can restrict the number of *bytes* or the number of *files*
 stored beneath that point in the directory hierarchy.
 
+Like most other things in CephFS, quotas are configured using virtual
+extended attributes:
+
+ * ``ceph.quota.max_files`` -- file limit
+ * ``ceph.quota.max_bytes`` -- byte limit
+
+If the extended attributes appear on a directory that means a quota is
+configured there. If they are not present then no quota is set on that
+directory (although one may still be configured on a parent directory).
+
+To set a quota, set the extended attribute on a CephFS directory with a
+value::
+
+  setfattr -n ceph.quota.max_bytes -v 100000000 /some/dir     # 100 MB
+  setfattr -n ceph.quota.max_files -v 10000 /some/dir         # 10,000 files
+
+To view quota limit::
+
+  $ getfattr -n ceph.quota.max_bytes /some/dir
+  # file: dir1/
+  ceph.quota.max_bytes="100000000"
+  $
+  $ getfattr -n ceph.quota.max_files /some/dir
+  # file: dir1/
+  ceph.quota.max_files="10000"
+
+.. note:: Running ``getfattr /some/dir -d -m -`` normally lists all the
```

Comment:
This is incorrect. We need to fix it anyway here.

The `getfatrr` won't list all the extended attribute, espically for the `security.selinux` extended attribute it won't.  For example:

```
# getfattr -d -m - file1111 
# file: file1111
user.long_attr="ddddd"

# getfattr -n 'security.selinux' --dump -e hex file1111 
# file: file1111
security.selinux=0x73797374656d5f753a6f626a6563745f723a6675736566735f743a733000

```

I will only list all the `user.` prefixed extended attributes.

Reply:
Okay. Fixed.